test/reality-capture.js

```diff
@@ -0,0 +1,70 @@
+const assert = require('assert');
+
+const { RealityCaptureClient } = require('..');
+
+describe('RealityCaptureClient', function() {
+    beforeEach(function() {
+        const { FORGE_CLIENT_ID, FORGE_CLIENT_SECRET, SCENE_NAME, SCENE_CALLBACK, SCENE_FORMAT, SCENE_TYPE} = process.env;
+        assert(FORGE_CLIENT_ID);
+        assert(FORGE_CLIENT_SECRET);
+        assert(SCENE_NAME);
+        assert(SCENE_CALLBACK);
+        assert(SCENE_FORMAT);
+        assert(SCENE_TYPE);
+        this.client = new RealityCaptureClient({ client_id: FORGE_CLIENT_ID, client_secret: FORGE_CLIENT_SECRET});
+        this.scenename = SCENE_NAME;
+        this.scenecallback = SCENE_CALLBACK;
+        this.sceneformat = SCENE_FORMAT;
+        this.scenetype = SCENE_TYPE;
+    });
+
+    describe('createPhotoScene()', function() {
+        it('should create a new photoscene', async function() {
+            const photoscene = await this.client.createPhotoScene(this.scenename, this.scenecallback, this.sceneformat, this.scenetype);
+            this.sceneid = photoscene.Photoscene.photosceneid;
+            assert(photoscene);
+        });
+    });
+
+    describe('addImages()', function() {
+        it('should add images to the photoscene', async function() {
+            const images = await this.client.addImages(this.sceneid, this.sceneformat, []);
```

Comment:
As discussed on Slack, I'm afraid referencing `this.sceneid` here isn't a good idea. Unit tests should never depend on the output of another test. Here you may need to hard-code the scene ID just like you're hard-coding the `SCENE_NAME` for example.

Reply:
Here's a discussion on the topic of "how do I (should I) make a test depend on another test": https://stackoverflow.com/questions/21704206/how-do-i-write-mocha-tests-where-one-tests-depends-on-the-result-for-the-next-te.

Reply:
For now, I'd suggest that you comment out any tests that create/update/delete content.

Reply:
I've updated the test file for reality-capture. Please review.